zzz_generated/app-common/app-common/zap-generated/enums.h

```diff
@@ -258,17 +258,6 @@ enum EmberAfSaturationStepMode : uint8_t
     EMBER_ZCL_SATURATION_STEP_MODE_DOWN = 3,
 };
 
-// Enum for SecurityType
-enum EmberAfSecurityType : uint8_t
-{
-    EMBER_ZCL_SECURITY_TYPE_UNSPECIFIED = 0,
-    EMBER_ZCL_SECURITY_TYPE_NONE        = 1,
-    EMBER_ZCL_SECURITY_TYPE_WEP         = 2,
-    EMBER_ZCL_SECURITY_TYPE_WPA         = 3,
-    EMBER_ZCL_SECURITY_TYPE_WPA2        = 4,
-    EMBER_ZCL_SECURITY_TYPE_WPA3        = 5,
-};
```

Comment:
This seems to be used by some of the ConnectivityManagerImpl and DragnosticsDataProviderImpl?

Reply:
Yep, I updated the PR to make them stop using that... which uncovered a bunch of bugs because the wrong integers were being assigned to the wrong things.